pkg/controllers/dynakube/logmonitoring/daemonset/annotations.go

```diff
@@ -0,0 +1,16 @@
+package daemonset
+
+import (
+	"github.com/Dynatrace/dynatrace-operator/pkg/api"
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
+)
+
+const annotationTenantTokenHash = api.InternalFlagPrefix + "tenant-token-hash"
+
+func (r *Reconciler) buildAnnotations(dk *dynakube.DynaKube) map[string]string {
```

Comment:
the name of method is too generic, you are doing concrete thing here. What about
```
annotationsWithTokenHash()
```

Reply:
That was on purpose as this place should as the one place where all the annotations are built, will make it more clear though